zk-cycle-utils/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "zk-cycle-utils"
```

Comment:
I think we should have a high level folder `utils` and inside a crate `zk-utils`. The way we have a place where we can to add common functionality. @preston-evans98 WDT?

Reply:
Sounds good to me!